src/components/property-detail/ListerDetails.js

```diff
@@ -0,0 +1,145 @@
+import { useState } from "react";
+import styled from "styled-components";
+import { Star } from '@styled-icons/boxicons-solid';
+
+const AgentPhotoUrl = styled.img`
+  border: 1px;
+  border-style: solid;
+  border-color: #D2D5DA;
+  border-radius: 50%;
+  height: 60px;
+  width: 60px;
+`;
+
+const AgentDetailsContainer = styled.div`
+  display: flex;
+  justify-content: flex-start;
+`;
+
+const AgentPhotoUrlContainer = styled.a`
+  margin: 18px 16px;
+`;
+
+const AgentDetails = styled.div`
+  margin: 16px 0;
+`;
+
+const AgentName = styled.div`
+  font-family: "PangeaRegular";
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #2B6ED2;
+`;
+
+const AgentNameLink = styled.a`
+  text-decoration: none;
+  &:visited {
+    color: #2B6ED2
+  };
```

Comment:
After moving the styles from `AgentName` in `AgentNameLink`, then we remove these three lines.